packages/here-wallet/src/lib/index.ts

```diff
@@ -20,6 +21,11 @@ export function setupHereWallet({
   defaultProvider,
 }: Options = {}): WalletModuleFactory<HereWallet> {
   return async () => {
+    const mobile = isMobile();
+
+    if (mobile) {
```

Comment:
This check is not needed for here wallet because it's not an extension wallet.

Reply:
We are a mobile wallet, but we can sign transactions on the desktop. Do not delete us 😅

Reply:
No worries, this will be reverted, that's why I left the above comment.